src/test/ValidatorKeys_test.cpp

```diff
@@ -136,60 +135,121 @@ class ValidatorKeys_test : public beast::unit_test::suite
             auto const keyType = KeyType::ed25519;
```

Comment:
One issue with storing `"public_key"` and not `"key_type"` is that you could replace the `"public_key"` in the file with a public key derived from the same secret key using a *different key type*, and the tool would have no way of knowing to complain.

I'm going to leave `["key_type"]` in the file, partly to future-proof and partly because the documentation no longer encourages the user to open/inspect the file.